torchvision/prototype/transforms/_augment.py

```diff
@@ -94,7 +94,7 @@ def _get_params(self, flat_inputs: List[Any]) -> Dict[str, Any]:
     def _transform(
         self, inpt: Union[datapoints.ImageType, datapoints.VideoType], params: Dict[str, Any]
     ) -> Union[datapoints.ImageType, datapoints.VideoType]:
-        if params["v"] is not None:
+        if params["v"] is not None:  # What is this?
```

Comment:
It's a value tensor or None used to erase the image

Reply:
Basically it is the replacement that is put in the "erased" area. In v1, in case we didn't find an area to erase, we return the bounding box of the whole image as well as the image

https://github.com/pytorch/vision/blob/01d138d81ae97b846ff63339eeee51f44a1725a2/torchvision/transforms/transforms.py#L1702

With that we call `F.erase` unconditionally, which ultimately leads to replacing every value in the original image with itself:

https://github.com/pytorch/vision/blob/01d138d81ae97b846ff63339eeee51f44a1725a2/torchvision/transforms/functional_tensor.py#L928-L932

Since that is quite nonsensical, we opted to also allow `None` as a return value and use it as a sentinel to do nothing. I think the previous implementation came from a time were JIT didn't support `Union` (or `Optional` for that matter) and thus we couldn't return `Optional[torch.Tensor]`.